sycl/test/check_device_code/id_queries_fit_int.cpp

```diff
@@ -1,4 +1,4 @@
-// RUN: %clangxx -fsycl -fsycl-id-queries-fit-in-int -O0 -c -S -emit-llvm -o %t.ll %s
+// RUN: %clangxx -fsycl -fsycl-id-queries-fit-in-int -disable-llvm-passes -O0 -c -S -emit-llvm -o %t.ll %s
```

Comment:
I think -O0 should be removed as it was added exactly for the same reason you added `-disable-llvm-passes`.
AFAIK, `-disable-llvm-passes` is not a driver option, so you can use it w/o `-Xclang`, but we have a SYCL specific option which should work for here - `-fno-sycl-early-optimizations`

I think the test is supposed to check device compiler output, so while you are here, please, replace `-fsycl` with `-fsycl-device-only`.

```suggestion
// RUN: %clangxx -fsycl-device-only -fsycl-id-queries-fit-in-int -fno-sycl-early-optimizations -S -o %t.ll %s
```

Reply:
Done.